charts/karpenter-crd/templates/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -940,6 +913,9 @@ spec:
                           RootVolume is a flag indicating if this device is mounted as kubelet root dir. You can
                           configure at most one root volume in BlockDeviceMappings.
                         type: boolean
+                    required:
+                      - deviceName
+                      - ebs
```

Comment:
I know this came from the new kubebuilder version, but this stands out as a bit of a red flag to me. Have we verified what the consequences of this change are?

Reply:
Good catch -- updated so we don't change the fields that are required here -- though the API may be invalid anyways without those fields being required so we may want to validate that in a subsequent PR